src/draw.c

```diff
@@ -433,6 +435,9 @@ static cairo_surface_t *render_background(cairo_surface_t *srf, colored_layout *
         }
 
         cairo_destroy(c);
+
+        *ret_width = width;
```

Comment:
possible `NULL` deref?

Albeit the function is not intended to get used without a return value of the width, it might be in future and save some debugging ;-)